diff-manifest.py

```diff
@@ -22,7 +22,11 @@ def main():
     after = get_manifest("HEAD")
 
     call("git", "fetch", "origin", "master:master")
-    merge_base = call("git", "merge-base", "master", "HEAD").strip()
+    commit_range = os.environ.get("TRAVIS_COMMIT_RANGE")
+    if commit_range:
+        merge_base = commit_range.split(".", 1)[0]
+    else:
+        merge_base = call("git", "merge-base", "master", "HEAD").strip()
```

Comment:
Oh it isn't, that's the main reason I actually stopped running this file. It should just be removed.